docs/pages/guides/prebuild.md

```diff
@@ -0,0 +1,168 @@
+
+**With Prebuild** the only side-effect is the Config Plugin entry in a project's Expo config (`app.json`), which will fail to evaluate when the corresponding package has been uninstalled, meaning it's nearly impossible to create orphaned configuration.
+
+## Anti Pitch
+
+Here are some reasons _Expo Prebuilding_ might **not** be the right fit for a particular project. There are also some notes on how we plan on alleviating the existing drawbacks eventually.
+
+### React Native versioning
+
+`npx expo prebuild` generates native code based on the version of `expo` a project has installed, so a project with SDK 46 (`expo@46.0.0`) would generate a `react-native@0.69.4` app.
+
+Expo has a quarterly release cycle, whereas `react-native` releases major versions at random intervals, with stability patches following a few weeks after. This means there are times where you cannot use `npx expo prebuild` with the latest release of React Native. This could potentially be circumvented by using a custom [Prebuild template](#templates), but this is not recommended.
+
+If React Native moves to a more predictable and stable release schedule then we could potentially add support during a beta release period.
+
+### Platform compatibility
+
+Prebuild can only be used for native platforms that are supported by the Expo SDK. This means iOS and Android for the time being. With the exception of web, which doesn't require `npx expo prebuild` since it uses the browser instead of a custom native runtime.
+
+We plan to support more platforms in the future, but team size and limited resources prevent us from being able to maintain them at the moment. The stability of out-of-tree platforms like `react-native-macos` and `react-native-windows` are also considered.
+
+You can still use prebuild for your `ios`, and `android` projects but any extra platforms will have to be configured manually in the meantime.
+
+### Standard native development is slower
+
+All native changes must be added with Expo Modules Core and Config Plugins. This means if you want to add some quick native file to your project the old fashion way, then your better off adding the file manually and working your way back into the system via a monorepo (we recommend [expo-yarn-workspaces](https://www.npmjs.com/package/expo-yarn-workspaces)). We plan to speed this process up by adding functionality to Expo Autolinking that finds project native files outside of the native folders and links them before building.
+
+If you want to do something like modifying the gradle properties file, you'll have to write a plugin for that [example](https://github.com/expo/expo/blob/1c994bb042ad47fbf6878e3b5793d4545f2d1208/apps/native-component-list/app.config.js#L21-L28). Of course this could be easily automated with helper plugin libraries, but it is a bit slower if you need to do it often.
```

Comment:
```suggestion
If you want to modify configuration, such as the **gradle.properties** file, you'll have to write a plugin ([example](https://github.com/expo/expo/blob/1c994bb042ad47fbf6878e3b5793d4545f2d1208/apps/native-component-list/app.config.js#L21-L28)). This can be easily automated with helper plugin libraries, but it is a bit slower if you need to do it often.
```